### PR TITLE
Period over period test

### PR DESCRIPTION
**Pull request by Marlene-M-Hirose**
Add the active_users_aggregation view and the PoP method. 

-still need to add country codes join

### CODE REVIEW

File: fenix/views/clients_counts_pop_method1.view.lkml [hunk 1]
```diff
@@ -0,0 +1,83 @@
+include: "//looker-hub/fenix/views/client_counts.view.lkml"
+
+
+view: clients_counts_pop_method1 {
+  extends: [client_counts]
+  dimension_group: first_seen_pop {
+    # sql: ${TABLE}.first_seen_date ;;
```
Comment (kik-kik) [inline fenix/views/clients_counts_pop_method1.view.lkml:7]: Is this line needed?

File: fenix/views/clients_counts_pop_method1.view.lkml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    type: time
+    view_label: "_PoP"
+    timeframes: [
+      raw,
```
Comment (kik-kik) [inline fenix/views/clients_counts_pop_method1.view.lkml:11]: is `raw` just original value? We can probably remove it? `date` is probably lowest granularity we'd want to look at?

File: fenix/views/clients_counts_pop_method1.view.lkml [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+                OR
+            (EXTRACT(DAYOFYEAR FROM ${first_seen_pop_raw}) = EXTRACT(DAYOFYEAR FROM CURRENT_DATE()) AND
+            EXTRACT(HOUR FROM ${first_seen_pop_raw}) < EXTRACT(HOUR FROM CURRENT_TIME()))
+                OR
+            (EXTRACT(DAYOFYEAR FROM ${first_seen_pop_raw}) = EXTRACT(DAYOFYEAR FROM CURRENT_DATE()) AND
+            EXTRACT(HOUR FROM ${first_seen_pop_raw}) <= EXTRACT(HOUR FROM CURRENT_TIME()) AND
+            EXTRACT(MINUTE FROM ${first_seen_pop_raw}) < EXTRACT(MINUTE FROM CURRENT_TIME())))  ;;
+  }
+
+  # measure: total_user_count {
```
Comment (kik-kik) [inline fenix/views/clients_counts_pop_method1.view.lkml:71]: Should this be removed?

File: kpi/views/unified_metrics_pop_method1.view.lkml [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+    label: "Total sum of user count"
+    view_label: "_PoP"
+    type: sum
+    sql: ${TABLE}.clients ;;
+    value_format_name: usd
+    drill_fields: [first_seen_date]
+  }
+
+
+  # measure: clients {
```
Comment (kik-kik) [inline kpi/views/unified_metrics_pop_method1.view.lkml:108]: is this needed?

File: kpi/views/unified_metrics_pop_method1.view.lkml [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+      quarter,
+      year,
+    ]
+    datatype: date
+    #add 1450 days as a test for shifting the old timestamp to recent timestamp
+    sql: timestamp_add(${TABLE}.first_seen_date, interval 1450 day) ;;
+    convert_tz: no
+  }
+
+  # (Method 1a) you may also wish to create MTD and YTD filters in LookML
```
Comment (kik-kik) [inline kpi/views/unified_metrics_pop_method1.view.lkml:47]: Should we maybe include a link to the docs for this method?

File: combined_browser_metrics/views/active_users_aggregates.view.lkml [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+  convert_tz: no
+  datatype: date
+  sql: ${TABLE}.submission_date ;;
+}
+
+dimension: pop_row  {
+  view_label: "_PoP"
+  label_from_parameter: choose_breakdown
+  type: string
+  # order_by_field: sort_hack1 # Important # WON: no dimension called sort_hack1
```
Comment (kik-kik) [inline combined_browser_metrics/views/active_users_aggregates.view.lkml:50]: Is this needed?

**Review by gkabbz — APPROVED** [review]
This pull request should have been split in two. One for methods and one for combined_browser_metrics. In the interest of time and to unblock the Analytics engineering workweek, I'll approve it so the explore for aggregate metrics is available for others to build. @Marlene-M-Hirose please submit a separate pull to either move the methods testing for period over period to looker development once back in office.

**Pull request merged**